include/sound/sof/stream.h

```diff
@@ -83,10 +83,11 @@ struct sof_ipc_stream_params {
 	uint16_t sample_valid_bytes;
 	uint16_t sample_container_bytes;
 
-	/* for notifying host period has completed - 0 means no period IRQ */
 	uint32_t host_period_bytes;
+	uint16_t period_irq; /* 1 means period IRQ mode ON */
 
-	uint32_t reserved[2];
+	uint16_t not_used;
+	uint32_t reserved[1];
```

Comment:
merge these 2 to "uint16_t reserved[3];"?

Reply:
`uint16_t reserved[5];`?

Reply:
Yes, it would be much nicer.